src/features/surveys/managers/surveyAnswerManager.ts

```diff
@@ -1,11 +1,11 @@
 import { useState, useEffect, useCallback } from 'react';
 import toast from 'react-hot-toast';
-import { LocalStorageKeys } from 'features/surveys/constants/types';
-import useLocalStorage from 'features/surveys/hooks/useLocalStorage';
+import { LocalStorageKeys } from '../ESS/src/features/surveys/constants/types';
+import useLocalStorage from '../ESS/src/features/surveys/hooks/useLocalStorage';
 import useTranslation from 'next-translate/useTranslation';
 import { useRouter } from 'next/router';
-import { getFetch, postFetch } from '../../../../lib/axiosConfig';
-import { SurveyWithQuestions } from 'types/SurveyWithQuestions';
+import { getFetch, postFetch } from '../../../../../ESS/lib/axiosConfig';
+import { SurveyWithQuestions } from '../ESS/src/types/SurveyWithQuestions';
```

Comment:
Why the imports are changed to strange `ESS/...` paths?

Reply:
Huh, that's really strange... i will fix that.